config/config_tests.xml

```diff
@@ -244,7 +244,12 @@ NODEFAIL          Tests restart upon detected node failure. Generates fake failu
     <BFBFLAG>TRUE</BFBFLAG>
     <STOP_OPTION>ndays</STOP_OPTION>
     <STOP_N>11</STOP_N>
+    <REST_N>$STOP_N / 2 + 1</REST_N>
```

Comment:
I disagree in this case: At least some of these settings are only supposed to apply to one case, so I find it confusing to see them set in the xml, which presumably makes them apply to both cases.